spec/components/file_store/s3_store_spec.rb

```diff
@@ -250,6 +251,13 @@
 
   end
 
+  it "s3_canned_acl" do
```

Comment:
What's the point of this test? If it's testing the validation, then it should at least test for a bad case.

Also, the validation is already tested in `s3_canned_acl_site_setting_spec.rb`, so not sure what this is testing.